docs/speech.rst

```diff
@@ -54,13 +45,24 @@ Functions
     that can be further hand-edited to improve accuracy, inflection and
     emphasis.
 
-.. py:function:: pronounce(phonemes, \*, pitch=64, speed=72, mouth=128, throat=128)
+.. py:function:: pronounce(phonemes, \*, pitch=64, speed=72, mouth=128, throat=128, pin=microbit.pin_speaker)
```

Comment:
Is the pin argument is V2 ?

Reply:
Yes. I was thinking it might be useful to provide an example for using the speaker pin as described in https://github.com/bbcmicrobit/micropython/pull/692/files/006755066013226a89382b9d59e84cdf1b63b8b6#diff-31fe7834cf8f5132fb12c807ca5d777bc29734a18d97f35b5050d0148ada3c3eR57

Do you think it should be left as it is?

Reply:
An example is good, but it's probably fine as long as it is explained in this section.

Reply:
I've left the example as is, as it doesn't sem to make sense to talk about an optional argument if we don't have an example in the function.